### PR TITLE
[2506] Sync BE errors with FE

### LINKED ISSUE #2506: Sync BE/FE errors
We need to sync the FE with the BE errors found here:
[openapi.yml](https://github.com/gnosis/gp-v2-services/blob/main/crates/orderbook/openapi.yml)

### PR DESCRIPTION
**Pull request by W3stside**
# Summary

Closes #2506 - syncs latest errors and descriptions with FE for appropriate handling in the UI

Screenshots
image

# To Test
Connect SC wallet
Set native as buy token
Input an amt as sell
Error should appear

### CODE REVIEW

**Review by elena-zh — APPROVED** [review]

**Comment by W3stside** [comment]
> Changes LGTM (according to the issue description)
> 
> However, I wonder why we show different messages on the FE from what I see in the files. As an example, 'Fees exceed from amount' image
> 
> Btw, there was a task to change this message on FE #1957

it's normal, its just the default description

**Comment by biocom** [comment]
On @elena-zh 's point, in my view the current text 'Fees exceed from amount' doesn't read natural. The `FeeExceedsFrom` text, reads more natural and provides better context. It's longer, but for that we need to adjust the UI.

File: src/custom/api/gnosisProtocol/errors/OperatorError.ts [hunk 1]
```diff
@@ -38,24 +44,30 @@ export enum ApiErrorCodeDetails {
...
+  NoLiquidity = 'Token pair selected has insufficient liquidity.',
   UnsupportedToken = 'One of the tokens you are trading is unsupported. Please read the FAQ for more info.',
-  AmountIsZero = 'Amount is zero',
-  SellAmountDoesNotCoverFee = 'Sell amount does not sufficiently cover the current fee',
-  TransferEthToContract = 'Sending the native currency to smart contract wallets is not currently supported',
+  AmountIsZero = 'Amount is zero.',
+  SellAmountDoesNotCoverFee = 'Sell amount does not sufficiently cover the current fee.',
+  TransferEthToContract = 'Sending native currency to smart contract wallets is not currently supported.',
+  TransferSimulationFailed = 'Transfer simulation of native currency failed. This is likely due to the current unsupported state of smart contract wallets.',
+  SameBuyAndSellToken = 'Sell and buy tokens cannot be the same.',
```
Comment (anxolin) [inline src/custom/api/gnosisProtocol/errors/OperatorError.ts:63]: token, in singular, right?

**Review by anxolin — APPROVED** [review]
Approve, but added a comment

File: src/custom/api/gnosisProtocol/errors/OperatorError.ts (hunk 1 above)
Comment (W3stside) [reply src/custom/api/gnosisProtocol/errors/OperatorError.ts:63]:
```suggestion
  SameBuyAndSellToken = 'Sell and buy token can not be the same.',
```

**Pull request merged**